src/vs/workbench/browser/parts/editor/editorCommands.ts

```diff
@@ -959,7 +962,7 @@ function registerCloseEditorCommands() {
 			}
 
 			// Make sure it becomes active too
-			await group?.openEditor(replacements[0].replacement);
+			await group.openEditor(replacements[0].replacement);
```

Comment:
I am not sure, shouldn't we make it active per group? I feel this entire action is not and probably should not be multi-group aware.

Reply:
It was already multi group aware for open editors view